repos/fdbt-site/src/pages/api/manageProductGroup.ts

```diff
@@ -0,0 +1,67 @@
+import { NextApiResponse } from 'next';
+import { getAndValidateNoc, redirectTo } from '../../utils/apiUtils';
+import { ErrorInfo, ManageProductGroupWithErrors, NextApiRequestWithSession } from '../../interfaces';
+import { getProductGroupByNameAndNocCode, insertProductGroup, updateProductGroup } from '../../data/auroradb';
+import { removeExcessWhiteSpace } from '../..//utils/apiUtils/validator';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { MANAGE_PRODUCT_GROUP_ERRORS_ATTRIBUTE } from '../../constants/attributes';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    const noc = getAndValidateNoc(req, res);
+    const productIdsFromReq: string[] | undefined | string = req.body.productsToExport;
+
+    const errors: ErrorInfo[] = [];
+    const id = req.body.id && Number(req.body.id);
+    const productGroupName = removeExcessWhiteSpace(req.body.productGroupName);
+    const isInEditMode = id && Number.isInteger(id);
+
+    if (productGroupName.length <= 2) {
+        errors.push({ errorMessage: 'Name cannot be less than 2 characters', id: 'product-group-name' });
+    }
+
+    if (!productIdsFromReq || productIdsFromReq.length === 0) {
+        errors.push({ errorMessage: 'Select product to be added in the group', id: '' });
+    }
+
+    let productIds: string[] = [];
+
+    if (typeof productIdsFromReq === 'string') {
+        productIds = [productIdsFromReq];
+    } else {
+        productIds = productIdsFromReq ? productIdsFromReq : [];
+    }
+
+    const productGroup = await getProductGroupByNameAndNocCode(noc, productGroupName);
+
+    if (productGroup !== undefined) {
```

Comment:
```suggestion
    if (productGroup) {
```
Check to see if something is defined liked this, or ``` !!productGroup```